test/test_functional_tensor.py

```diff
@@ -472,6 +472,55 @@ def test_resize(self):
             with self.assertRaisesRegex(ValueError, "max_size = 32 must be strictly greater"):
                 F.resize(img, size=32, max_size=32)
 
+    def test_resize_antialias(self):
+
+        if self.device == "cuda":
+            self.skipTest("Not implemented for CUDA device")
```

Comment:
I have introduced a `@needs_cuda` decorator here https://github.com/pytorch/vision/pull/3792/files#diff-f833e6bb21df531837a51e84306266c2f1f6d1565340a498095868e24b3f27deR432 but it's not merged yet and it will only work with pytest tests.

It's safe to return instead of skipping here, but I would **really** encourage new tests to use pytest instead of UnitTest. In this particular case, it would be very helpful to re-write the test so that it uses `@pytest.mark.parametrize` instead of deeply-nested for-loops

See for example: https://github.com/pytorch/vision/blob/master/test/test_functional_tensor.py#L958:L959

Reply:
> I have introduced a @needs_cuda decorator here #3792 (files) but it's not merged yet and it will only work with pytest tests.

Actually this is useless here: if the test relies on pytest, there will be no need to skip anything. The test won't run on GPU since it won't inherit from CudaTester. Another reason to port the test to `pytest` :)

Reply:
Side note: This will however be a good candidate for the `@only_needs_cpu` decorator so that this test doesn't run on CI GPU machines (to save time and money) but I haven't introduced it yet

Reply:
Rewritten the test using `pytest.mark.parametrize`.